test/TestProjects/MgmtListOnly/Generated/AvailabilitySet.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.ResourceManager.Core;
+using MgmtListOnly.Models;
+
+namespace MgmtListOnly
+{
+    /// <summary> A Class representing a AvailabilitySet along with the instance operations that can be performed on it. </summary>
+    public class AvailabilitySet : AvailabilitySetOperations
```

Comment:
I prefer if we use non real names since it can get confusing if you know what an AvailabilitySet should look like.  I realize we haven't been completely consistent in this, but I would like to nudge in that direction if possible.

Reply:
Let me rewrite the test swagger to use made-up names so that we would not confuse these with real life things

Reply:
It is done